bugbot/utils.py

```diff
@@ -765,3 +765,28 @@ def is_keywords_removed_by_bugbot(bug: dict, keywords: Iterable) -> bool:
         if change["field_name"] == "keywords"
         for keyword in keywords
     )
+
+
+def get_bug_bugdash_url(component, tab_name: str) -> str:
+    """
+    Generate bugdash URL from a component.
+
+    Args:
+        component: The component as a ComponentName object.
+        tab_name: Specifies the tab of the component.
+
+    Returns:
+        Bugdash URL for the component.
+    """
+    # Bugdash uses a single colon instead of a double colon to prefix the product name.
+    encoded_component = quote_plus(f"{component.product}:{component.name}")
+
+    valid_tabs = ["triage", "important", "stalled", "reo", "tracked"]
```

Comment:
I do not think it is necessary to validate the tab names. Also, this could become outdated if a new tab gets added.